README.md

```diff
@@ -7,6 +7,14 @@ This repository contains examples of microservices implemented using
 the directory name. The [cellar](https://github.com/goadesign/examples/tree/master/cellar) example
 provides a complete implementation of a simple microservice.
 
+The samples in each directory serve as templates, and you can clone them using the `gonew` command, as explained in [this blog post](https://go.dev/blog/gonew).
+
+```shell
+$ go install golang.org/x/tools/cmd/gonew@latest
+$ gonew github.com/ikawaha/examples/basic@latest github.com/<your_repo>/basic
```

Comment:
should this be `gonew goa.design/examples/basic@latest github.com/<your_repo>/basic`?

Reply:
ooops, fixed in https://github.com/goadesign/examples/pull/130/commits/e0823af5012140691602c7e782994fe488c4a6e0